pkg/virtual/syncer/builder/template.go

```diff
@@ -55,7 +55,7 @@ type templateProvider struct {
 	kubeClusterClient    kcpkubernetesclientset.ClusterInterface
```

Comment:
Should we do something with this as well ? It is used to call the SAR in the authorizer, and we need replicated service accounts here (cf [this discussion](https://kubernetes.slack.com/archives/C021U8WSAFK/p1675252445541419?thread_ts=1675251008.266109&cid=C021U8WSAFK) and followings).
But to work this will require the work from @s-urbaniak about replicating some serviceaccounts.

Reply:
We don't need to vary this client. It's always talking to its shard.